cosmos-auth/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cosmos-auth",
   "description": "Node.js app exposing Cosmos Auth API",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/0026c17b38708be56dd7c986e84f62ddb02df7cd